server/src/api_endpoints/personas/utils.js

```diff
@@ -70,10 +70,35 @@ const checkRoleExists = async (roleKey) => {
   }
 }
 
+/**
+ * Converts a filter query string into a filter object
+ *
+ * @param {string} filterQuery - The filter query string (E.g. 'tenantname:value1, sub_claim:value2')
+ * @returns {Object} - The generated filter object
+ * @throws {Object} - Error object with HTTP status code and message if the filter query is invalid
+ */
+const getFilterObjectFromFilterQueryString = (filterQuery) => {
```

Comment:
@k1ch  assuming that we need to keep this function based on my other comments, should this function have unit tests to cover it separate from the personas API tests?